server/base/src/main/java/org/apache/accumulo/server/rpc/TServerUtils.java

```diff
@@ -186,7 +187,43 @@ public static ServerAddress startServer(ServerContext service, String hostname,
   }
 
   /**
-   * Create a NonBlockingServer with a custom thread pool that can dynamically resize itself.
+   * Create a non blocking server with multiple select threads and a custom thread pool that can
+   * dynamically resize itself.
+   */
+  public static ServerAddress createThreadedSelectorServer(HostAndPort address,
```

Comment:
Does this server type support SSL or SASL? If so, we could consolidate server implementations a bit.

Reply:
I have no idea.